sdk/test/AzureTest/DVRSessionTransport.swift

```diff
@@ -64,8 +64,14 @@ public class DVRSessionTransport: TransportStage {
         }
         session = Session(outputDirectory: outputDirectory, cassetteName: cassetteName)
         if environmentVariable(forKey: "TEST_MODE", default: "playback") == "record" {
+            session?.recordMode = .newEpisodes
```

Comment:
If in record mode, this should be `.all`. I don't think we'll use `.once` or `.newEpisodes` at all.